website/src/components/canvas/Sidebar.jsx

```diff
@@ -89,7 +112,18 @@ class Sidebar extends React.Component {
   mouseEnter() {
     this.setState({opacity: 1})
   }
-
+  onClickClose() {
+   this.setState(prevState => ({
+     isOpen: !prevState.isOpen,
+     closed: 1
+   }));
+ }
+ onClickOpen() {
```

Comment:
I understand what you're saying with the naming ,so I'll change that. However, the syntax is based off of ES6 standards that Facebook talks about here: https://facebook.github.io/react/docs/handling-events.html ,so I highly doubt it will glitch out ....but it makes sense to do it from a readibility standpoint